### ISSUE TITLE
Step 4 installation problem

### ISSUE THREAD

**Issue by ConnieXuhm** [weizhouUMICH/SAIGE#257]
Hi, when I conduct step 4, "devtools::install_github("weizhouUMICH/SAIGE")", it happens like this:
sh: 1: /bin/gtar: not found
sh 2: /bin/gtar: not found
Error: failed to install from github
In additition: warning messages:
1: In system(cmd): error in running command
2: in utils:: untar (tarfile,...):
 "/bin/gtar -xf '/tmp/Rtmpqtv4kd/file5b47c239d71.tar.gz' ...error code 127

Could you help me with this? thx!

**Comment by matuskosut** [comment]
@ConnieXuhm it fail because R devtools is not able to find gtar, usually applying normal tar before running devtools::install* helps:
```
Sys.setenv(TAR = "/bin/tar")
```

just make sure that you have tar: `file /bin/tar` or find it with: `which tar`

**Comment by ConnieXuhm** [comment]
Thanks @matuskosut ! But I met the following problem:
Failed to build package statgen/savvy
invalid lock file in /home/guest/anaconda3/bin/SAIGE/thirdParty/bgen
Setting top to                           : /home/guest/anaconda3/bin/SAIGE/thirdParty/bgen 
Setting out to                           : /home/guest/anaconda3/bin/SAIGE/thirdParty/bgen/build 
Checking for 'gcc' (C compiler)          : /home/guest/anaconda3/envs/RSAIGE8/bin/x86_64-conda_cos6-linux-gnu-cc 
Checking for 'g++' (C++ compiler)        : /home/guest/anaconda3/envs/RSAIGE8/bin/x86_64-conda_cos6-linux-gnu-c++ 
zlib                                     : not found 
The configuration failed
(complete log in /home/guest/anaconda3/bin/SAIGE/thirdParty/bgen/build/config.log)
invalid lock file in /home/guest/anaconda3/bin/SAIGE/thirdParty/bgen
The project was not configured: run "waf configure" first!
ERROR: configuration failed for package ‘SAIGE’

And I look back in config.log, there is an error:
 'testprog' failed with exit status 1

 How can I fix with this ? Thanks again!

**Comment by matuskosut** [comment]
@ConnieXuhm  I see that you are using conda

First problem that I see is `Failed to build package statgen/savvy`, for this in your main environment I recommend installing cget, because saige is using cget to download other dependencies, e.g.:
```
conda config --add channels conda-forge
conda install -n root -c conda-forge cget
```

the other one is `zlib : not found`, have you tried to make sure zlib is installed? `conda install -c conda-forge zlib`
not sure if you also read advices here: https://github.com/weizhouUMICH/SAIGE/issues/118

if it still fails, you should post your env export to see which versions of packages you have: `conda env export`, linking this export here using [Gist](http://gist.github.com/) might be cleaner..

**Comment by ConnieXuhm** [comment]
Much Thanks to @matuskosut  ! I successfully install SAIGE and move on to the next step.

**Comment by weizhouUMICH** [comment]
We have just released a new version 1.0.0. It has computational efficiency improvements for both Step 1 and Step 2 for single-variant and set-based tests. We have created a new program github page https://github.com/saigegit/SAIGE with the documentation provided https://saigegit.github.io/SAIGE-doc/
The program will be maintained by multiple SAIGE developers there. The docker image has been updated. Please feel free to try the version 1.0.0 and report issues if any.

Thanks!
Wei
